plugins/serializers/nowmetric/nowmetric.go

```diff
@@ -69,7 +69,8 @@ func (s *serializer) createObject(metric telegraf.Metric) ([]byte, error) {
 	/*  ServiceNow Operational Intelligence supports an array of JSON objects.
 	** Following elements accepted in the request body:
 		 ** metric_type: 	The name of the metric
-		 ** resource:   	Information about the resource for which metric data is being collected. In the example below, C:\ is the resource for which metric data is collected
+		 ** resource:   	Information about the resource for which metric data is being collected.
+	                        In the example below, C:\ is the resource for which metric data is collected
```

Comment:
Can we indent these the same?
```suggestion
		 ** resource:   	Information about the resource for which metric data is being collected.
	                                In the example below, C:\ is the resource for which metric data is collected
```

Reply:
Done.